### PR TITLE
Fix bootstrap-osg-test (SOFTWARE-2232)

### PR DESCRIPTION
**Pull request by brianhlin**
The OSG release ver gets populated by positional arg with these changes. Bonus commit for minimizing the git clone sizes

### CODE REVIEW

**Comment by brianhlin** [comment]
@timtheisen ready for re-review. See [step 6 of this doc](https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/) for how to submit suggestions

File: bootstrap-osg-test [hunk 1]
```diff
@@ -13,6 +13,20 @@ on_error_die()
     fi
 }
 
+if [ $# -lt 1 ]; then
```
Comment (timtheisen) [inline bootstrap-osg-test]:
```suggestion
if [ $# -ne 1 ]; then
```

File: bootstrap-osg-test [hunk 2]
```diff
@@ -13,6 +13,20 @@ on_error_die()
...
+    echo "Requires the major version number, i.e. 3.2 for 3.2.0"
+    echo "usage: $0 <major version> [testing|development]"
```
Comment (timtheisen) [inline bootstrap-osg-test]:
```suggestion
    echo "usage: $0 <major version>"
```

File: bootstrap-osg-test [hunk 3]
```diff
@@ -13,6 +13,20 @@ on_error_die()
...
+    exit 1
+fi
+
+if [[ $1 =~ ^[0-9]+\.[0-9]+$ ]]; then
+    version=$1
+else
+    echo "Major version should be in the form of '<number>.<number>, i.e. 3.2"
+    echo "usage: $0 <major version> [testing|development]"
```
Comment (timtheisen) [inline bootstrap-osg-test]:
```suggestion
    echo "usage: $0 <major version>"
```

**Pull request merged**